packages/office-ui-fabric-react/src/components/Facepile/Facepile.tsx

```diff
@@ -73,6 +74,7 @@ export class Facepile extends React.Component<IFacepileProps, {}> {
       onClick={ this._onPersonaClick.bind(this, persona) }
       onMouseMove={ this._onPersonaMouseMove.bind(this, persona) }
       onMouseOut={ this._onPersonaMouseOut.bind(this, persona) }
+      ref={ (el: HTMLElement) => this._refHandler(persona, el) }
```

Comment:
I think we should stick to following the event pattern, where the native thing comes first, so if we proceed it should be this.refHandler(el, persona)